lib/rucio/core/oidc.py

```diff
@@ -1405,7 +1404,7 @@ def validate_jwt(json_web_token: str, *, session: "Session") -> dict[str, Any]:
             clprocess = subprocess.Popen(['curl', '-s', '-L', '-u', '%s:%s'
                                           % (oidc_client.client_id, oidc_client.client_secret),
                                           '-d', 'token=%s' % (json_web_token),
-                                          oidc_client.introspection_endpoint],
+                                          oidc_client.introspection_endpoint],  # type: ignore
```

Comment:
note: this one is pending discussion on Mattermost to decide whether we can delete this whole if block or not

Reply:
discussed - best to ignore / avoid changing anything OIDC-related